libraries/testing/include/eosio/testing/tester.hpp

```diff
@@ -67,6 +67,8 @@ namespace eosio { namespace testing {
       full
    };
 
+   std::ostream& operator<<(std::ostream& os, setup_policy p);
```

Comment:
Maybe you can add a comment here saying its purpose is for boost data.

Reply:
Added comment.